static/js/invite.js

```diff
@@ -190,6 +192,15 @@ export function launch() {
 }
 
 export function initialize() {
+    const rendered = render_invite_user({
+        is_admin: page_params.is_admin,
+        is_owner: page_params.is_owner,
+        development_environment: page_params.development_environment,
+        invite_as_options: settings_config.user_role_values,
+    });
+
+    $(".app").append(rendered);
```

Comment:
If/when we want to support live-updating the current user's role, we'll want to migrate this modal to render on demand rather than on page load.  Feel free to do that as a follow-up while we're thinking about this, but I don't think it's required.

Reply:
Do you mean the initial value of dropdown here to be set as the referrer role?